README.md

```diff
@@ -115,7 +115,7 @@ Tools and packages you need for development:
 You can then build and install the app by cloning this repository into the Nextcloud apps folder and running `make build`.
 ```bash
 cd /var/www/<NEXTCLOUD_INSTALL>/apps
-git clone https://github.com/R0Wi/nextcloud_workflow_ocr.git workflow_ocr
+git clone https://github.com/R0Wi/workflow_ocr.git workflow_ocr
```

Comment:
The third command line argument `workflow_ocr` is not necessary because the repository has now been renamed and equals to the NC app name. So you can optionally omit the specified local directory of the Git clone command.
 
```suggestion
git clone https://github.com/R0Wi/workflow_ocr.git
```